src/rtcmix/RTcmixMain.h

```diff
@@ -42,6 +44,7 @@ class RTcmixMain : public RTcmix {
 	#endif
 	/* for more than 1 socket, set by -s flag to CMIX as offset from MYPORT */
 	static int		socknew;
+        static UdpListeningReceiveSocket *udpRcvSocket;
```

Comment:
Because UdpListeningReceiveSocket is a class, you can just forward-declare it just above the declaration of class RTcmixMain:

class UdpListeningReceiveSocket;

Then you will not need to include the new header here.